packages/klarna-integration/src/klarnav2/klarnav2-payment-strategy.ts

```diff
@@ -257,4 +262,25 @@ export default class KlarnaV2PaymentStrategy {
             );
         });
     }
+
+    private async klarnaOrderInitialization(methodId: string): Promise<void> {
```

Comment:
let's move this method to the [klarnav2-token-updater.ts](https://github.com/bigcommerce/checkout-sdk-js/blob/master/packages/klarna-integration/src/klarnav2/klarnav2-token-updater.ts)
and rename to something like `setKlarnaToken`